src/objects.js

```diff
@@ -1,41 +1,55 @@
 const createPerson = (name, age) => {
-  // your code here
+<<<<<<< HEAD
```

Comment:
You've committed a merge conflict here. I'd take these markers out and re-commit.